pkgs/development/python-modules/pylibrespot-java/default.nix

```
@@ -0,0 +1,31 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+
+  aiohttp,
+}:
+
+buildPythonPackage rec {
+  pname = "pylibrespot-java";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "uvjustin";
+    repo = "pylibrespot-java";
+    tag = "v${version}";
+    hash = "sha256-aPmyYsO8yBrlPEQXOGNjZvuO8QZr13SOH09gqjW4WPA=";
+  };
+
+  propagatedBuildInputs = [ aiohttp ];
```

```suggestion
  build-system = [ ];

  dependencies = [ aiohttp ];
```
you probably need to add setuptools to build-system